src/parcels/mount-parcel.js

```diff
@@ -155,6 +172,8 @@ export function mountParcel(config, customProps) {
     mountPromise: promiseWithoutReturnValue(mountPromise),
     unmountPromise: promiseWithoutReturnValue(unmountPromise),
   };
+
+  return externalRepresentation
```

Comment:
I must be misunderstanding something. As far as I can tell the current external representation doesn't expose the ability to call update, but it looks like that's working here: https://github.com/CanopyTax/single-spa/pull/198/files#diff-91ce6f9d0f3469d16f4cbba33b114f9dR75 I think I'm missing something.

Reply:
It isn't set immediately, but once the loadPromise is resolved (since the parcelConfig isn't always available immediately now that we can have parcels use loading functions).